src/mon/Monitor.cc

```diff
@@ -2766,7 +2766,8 @@ void Monitor::get_cluster_status(stringstream &ss, Formatter *f)
       for (set<int>::iterator p = quorum.begin(); p != quorum.end(); ++p)
 	f->dump_string("id", monmap->get_name(*p));
       f->close_section();
-      f->dump_stream("quorum_age") << (now - quorum_since);
+      f->dump_float("quorum_age",
```

Comment:
how about using `dump_int(duration_cast<seconds>((now - quorum_since).count());` to be explicit that we are printing the quorum_age in seconds. please note this change also prints the seconds in "int". because `std::chrono::seconds::rep` is a signed integer. but i don't think it matters in this context.